databricks/koalas/base.py

```diff
@@ -941,15 +943,21 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
         Name: koalas, dtype: int64
         """
         from databricks.koalas.series import Series, _col
+        if LooseVersion(pyspark.__version__) < LooseVersion("2.4") and \
+                default_session().conf.get("spark.sql.execution.arrow.enabled") == "true":
+            raise RuntimeError("if you're using pyspark < 2.4, set conf "
+                               "'spark.sql.execution.arrow.enabled' to 'false' "
+                               "for using this function")
```

Comment:
What's this for?

Reply:
A build test in travisCI has failed when python 3.5. (https://travis-ci.com/databricks/koalas/jobs/266345663) like below.

<img width="1176" alt="스크린샷 2019-12-13 오전 10 33 27" src="https://user-images.githubusercontent.com/44108233/70762486-071e1a00-1d94-11ea-9616-7e34aaad4508.png">

so i checked, and it seems there was an issue that pyspark2.3 doesn't support manage `spark.sql.execution.arrow.enabled` automatically for struct type. (maybe pyspark2.4 manage this conf automatically)

So, this for raising proper exception message about that.

Reply:
I see. The limitation is only when `MultiIndex`, right? Then we should throw only when `MultiIndex`.

Reply:
@ueshin ah, that makes sense. okay i'll fix it. thanks! :)